src/actionCreators/resourceHelpers.js

```diff
@@ -214,6 +209,28 @@ export const recursiveResourceFromDataset =
       )
     )
 
+const newSubjectFromDataset =
```

Comment:
Need to add the classes from the dataset to the subject model in state. Previously, the class from the subject template was used.

Reply:
should this be a comment?  "Need to add the classes from the dataset to the subject model in state"